python/cudf/cudf/core/column/column.py

```diff
@@ -1499,6 +1428,7 @@ def _mask_from_cuda_array_interface_desc(desc):
                 dtype=mask_dtype,
                 finalizer=None,
             )
+            # TODO: this can be done more efficiently
```

Comment:
Same as above, pass object to function instead of just the dictionary.

Reply:
Done